### ISSUE TITLE
Improve figure docstrings and name 

### ISSUE THREAD

**Issue by natema** [natema/WorldDynamics.jl#118]
The current docstrings reference a chapter without referring to a specific book. 
With the current plan to add more models, it would be better to mention the book explicitly, possibly as a linked acronym ([[DGFW](https://archive.org/details/dynamicsofgrowth0000unse)]):
```
WorldDynamics.World3.Agriculture.fig_69c
— Method

Reproduce Fig 4.69c. The original figure is presented on Chapter 4.
```

Also, we should consider changing the figure names to match the numbering in the book.

**Comment by natema** [comment]
@aurorarossi is also suggesting adding the original caption in the docstring.

**Comment by paulobruno** [comment]
> The current docstrings reference a chapter without referring to a specific book. With the current plan to add more models, it would be better to mention the book explicitly, possibly as a linked acronym ([[DGFW](https://archive.org/details/dynamicsofgrowth0000unse)]):
> 
> ```
> WorldDynamics.World3.Agriculture.fig_69c
> — Method
> 
> Reproduce Fig 4.69c. The original figure is presented on Chapter 4.
> ```
> 
> Also, we should consider changing the figure names to match the numbering in the book.

What do you mean by "match the numbering in the book"?  Aren't we are already following the same numbering?

**Comment by natema** [comment]
> What do you mean by "match the numbering in the book"? Aren't we are already following the same numbering?

E.g. `Fig 7-7` is called `fig_7` rather than `fig_7_7`.

**Comment by paulobruno** [comment]
> E.g. `Fig 7-7` is called `fig_7` rather than `fig_7_7`.

Ok, I see. This was informally discussed with @piluc and we agreed that using, for example, `World3.Agriculture.fig_4-69a` is redundant, since Agriculture is always Chapter 4.

**Comment by natema** [comment]
I agree, and the if the reader doesn't know that fact and open the book and wants to reproduce figure 4-69a, they can still find the command in the documentation, but I'm wondering if we could make it a bit more user friendly with little effort. 
E.g., we could define alias functions `World3.fig_4_69a() = World3.Agriculture.fig_4-69a()`. 
But I should have probably avoided mentioning this other point in this issue: the main one is to improve the docstrings.

**Comment by paulobruno** [comment]
Ok, so my suggestion is to use this issue only for the docstrings, without changing the names. Then we can start a [discussion](https://github.com/natema/WorldDynamics.jl/discussions) about the "alias" functions.

**Comment by paulobruno** [comment]
Fixed by #124
